.gitignore

```diff
@@ -17,4 +17,5 @@ tmp
 build_and_publish
 .rubocop*
 .DS_Store
-.ruby-version
\ No newline at end of file
+.ruby-version
+.idea
```

Comment:
```suggestion
```
I don't think we need to add this to the `.gitignore`

Reply:
done.